methplotlib/annotation.py

```diff
@@ -120,8 +121,8 @@ def assign_colors_to_genes(transcripts):
 
 def parse_bed(bed, window):
     logging.info("Parsing BED file")
-    df = pd.read_csv(bed, sep="\t", names=['chromosome', 'begin', 'end', 'name', 'score', 'strand'])
-    return df.loc[df['begin'].between(window.begin, window.end)
-                  | df['end'].between(window.begin, window.end)] \
-        .drop(columns=['chromosome', 'score', 'strand']) \
-        .itertuples(index=False, name=None)
+    gr = pr.read_bed(bed)[window.chromosome, window.begin:window.end]
+    df = gr.unstrand().df
+    df = df.drop(columns="Chromosome Score".split())
+    return df.itertuples(index=False)
```

Comment:
I used to have `df.itertuples(index=False, name=None)` to get regular tuples and not a named tuple, or is your solution equivalent? I don't need a name, and the default name is 'Pandas' (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.itertuples.html)

Reply:
Did not know about that. Thanks! Added `name=False` back :)